codemaster-jojo/Lost-in-Time
# Audio manager

### Audio Manager

Description:
- sounds array with Sound class from Sound script
- Sound class stores for each sound effect: name(string), clip(audio clip), volume, pitch, loop(bool)
- Play function in AudioManager runs sound effect from the sounds array

How to use/test:
- Add new sound effects to the sounds array by pressing the "+" symbol in the inspector of AudioManager game obj
   -  Set details/variables there as well
   - To add new audio clip, there is already a Sounds folder with many .mp3 files. Simply drag them into the "Clip" slot
- As of now, there is a AudioTest.cs script for testing
- Pressing "Space" and "W" plays a sound effect
- call the Play function(check script)
- Instructions are also in the script
- **Try adding your own sound effect in the inspector/script!**
